<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Edit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>271</width>
     <height>207</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_printTextEdit">
      <property name="text">
       <string>Print Text Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clearTextEdit">
      <property name="text">
       <string>Clear Text Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_setFont">
      <property name="text">
       <string>Set Font</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_setFontItalic">
      <property name="text">
       <string>Set Font Italic</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_setFontColor">
      <property name="text">
       <string>Set Font Color Red</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_fontSizeUp">
        <property name="text">
         <string>Font Size Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fontSizeDown">
        <property name="text">
         <string>Font Size Down</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textedit_Test">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>369</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
